apps/docs/pages/docs/integrating-puck/categories.mdx

```diff
@@ -33,14 +33,14 @@ const config = {
 };
 ```
 
-You can also change the label, collapse and hide categories:
+You can also change the title, collapse and hide categories:
 
 ```tsx {5,6,10} copy showLineNumbers
 const config = {
   categories: {
     typography: {
       components: ["HeadingBlock", "ParagraphBlock"],
-      label: "Text",
+      title: "Text",
```

Comment:
Just added this extra small fix to the docs that I noticed was wrong.

Reply:
Ah nice one ty